i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.md

```diff
@@ -15,76 +15,77 @@ Pour faire des demandes d'évolutions ou reporter des bugs sur les extensions co
 
 ## Centreon MAP
 
-### 22.10.0
+### 23.04.0
 
-Release date: `October 26, 2022`
+Release date: `April 26, 2023`
 
-- Compatibility with other 22.10 components.
-
-#### Performance
-
-- Move to Java 17
+- Compatibility with other 23.04 components.
 
 ## Centreon BAM
 
-### 22.10.0
+### 23.04.0
 
-Release date: `October 26, 2022`
+Release date: `April 26, 2023`
 
-- Compatibility with other 22.10 components.
+- Compatibility with other 23.04 components.
+- [Configuration] Removed deprecated "Import SSV" feature from Business Activities configuration.
+- [Configuration] Improved Business Activities configuration panel display:
+   - Display listing pagination icons while displaying the configuration panel
+   - Allow more space to display indicators' names.
 
 ## Centreon MBI
 
-### 22.10.0
-
-Release date: `October 26, 2022`
-
-- Compatibility with other 22.10 components.
+### 23.04.0
 
-#### Performance
+Release date: `April 26, 2023`
 
-- ETL daily and rebuild optimizations
-- Move to Java 17
+- Compatibility with other 23.04 components.
 
 ## Centreon Auto Discovery
 
-### 22.10.0
+### 23.04.0
 
-Release date: `October 26, 2022`
+Release date: `April 26, 2023`
 
-- Compatibility with other 22.10 components.
+- Compatibility with other 23.04 components.
+- Plugins automatic installation: plugins are now installed automatically on the poller where a Host Discovery Job is scheduled. So, you won't need to install them prior to scheduling discovery jobs.
+- Job duplication: Host Discovery jobs can now be duplicated to avoid repetitive work.
+- Mapper duplication: Host Discovery Mappers can now be duplicated inside a job.
 
-#### Enhancements
+## Centreon Monitoring Connectors Manager
```

Comment:
@cg-tw should we not add "(formerly Plugin Packs Manager)" to the title for this release?